src/light_data/mod.rs

```diff
@@ -156,6 +168,10 @@ impl LightData {
     }
 
     /// Deserializes a `LightData` from a `&[u8]`.
+    ///
+    /// # Errors
+    ///
+    /// This function errors if the input bytes don't correspond to a valid serialization of LightData
```

Comment:
I think this comment is inaccurate, since `de_aux()` is called and can panic, right?

Reply:
Same